Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -61,6 +61,17 @@ LogManager::LogManager()
   m_Log[LogTypes::GDB_STUB] = new LogContainer("GDB_STUB", "GDB Stub");
   m_Log[LogTypes::GPFIFO] = new LogContainer("GP", "GPFifo");
   m_Log[LogTypes::HOST_GPU] = new LogContainer("Host GPU", "Host GPU");
+  m_Log[LogTypes::IOS] = new LogContainer("IOS", "IOS");
+  m_Log[LogTypes::IOS_DI] = new LogContainer("IOS_DI", "IOS - Drive Interface");
+  m_Log[LogTypes::IOS_ES] = new LogContainer("IOS_ES", "IOS - ETicket Services");
+  m_Log[LogTypes::IOS_FILEIO] = new LogContainer("IOS_FILEIO", "IOS - FileIO");
+  m_Log[LogTypes::IOS_HID] = new LogContainer("IOS_HID", "IOS - USB_HID");
+  m_Log[LogTypes::IOS_SD] = new LogContainer("IOS_SD", "IOS - SDIO");
+  m_Log[LogTypes::IOS_SSL] = new LogContainer("IOS_SSL", "IOS - SSL");
+  m_Log[LogTypes::IOS_STM] = new LogContainer("IOS_STM", "IOS - State Transition Manager");
+  m_Log[LogTypes::IOS_NET] = new LogContainer("IOS_NET", "IOS - Network");
+  m_Log[LogTypes::IOS_WC24] = new LogContainer("IOS_WC24", "IOS - WiiConnect24");
+  m_Log[LogTypes::IOS_WIIMOTE] = new LogContainer("IOS_WIIMOTE", "IOS - Wiimote");
```

Comment:
The logs themselves aren't translatable (they're way too many, too technical, and rarely useful for users), and since those aren't translatable, there's not much point in making these other logging-related strings translatable either.